coala_quickstart/generation/Settings.py

```diff
@@ -92,11 +92,11 @@ def generate_settings(project_dir, project_files, ignore_globs, relevant_bears):
         [ext for lang in lang_files for ext in extset[lang]],
         relevant_bears[lang_map["all"]])
 
-    settings["default"]["ignore"] = generate_ignore_field(
-        project_dir,
-        lang_files.keys(),
-        extset,
-        ignore_globs)
+    str = generate_ignore_field(project_dir, lang_files.keys(), extset,
```

Comment:
plus str may or many not be a annotation name, so really not a good idea to use it